packages/portfolio-contract/src/type-guards.ts

```diff
@@ -223,6 +227,10 @@ type FlowStatus = {
   error?: string;
 };
 
+/** ChainNames including those in future upgrades */
+type ChainNameExt = string;
+const ChainNameExtShape: TypedPattern<ChainNameExt> = M.string();
```

Comment:
It's not really any arbitrary string, right? I get that we want an open set so adding one doesn't require a contract upgrade but it would be good to at least illustrate some examples.

Perhaps define the known ones for the static type? That's easy enough to revise.